packages/arb-token-bridge-ui/src/hooks/TransferPanel/useGasEstimates.ts

```diff
@@ -1,49 +1,74 @@
 import { BigNumber } from 'ethers'
-import { Provider } from '@ethersproject/providers'
+import { StaticJsonRpcProvider } from '@ethersproject/providers'
 import useSWR from 'swr'
 
-import { GasEstimates } from '../arbTokenBridge.types'
+import { DepositGasEstimates, GasEstimates } from '../arbTokenBridge.types'
 import { depositTokenEstimateGas } from '../../util/TokenDepositUtils'
 import { withdrawInitTxEstimateGas } from '../../util/WithdrawalUtils'
 import { Address } from '../../util/AddressUtils'
 import { depositEthEstimateGas } from '../../util/EthDepositUtils'
+import { isNetwork } from '../../util/networks'
 
-type TransferType = 'deposit' | 'withdrawal'
+async function isDepositMode({
```

Comment:
`isDepositMode` is returned by `useNetworksRelationship` already

Reply:
We can either pass `isDepositMode` from the hook (because fetcher can't call hook directly), or we can reuse the logic for `isDepositMode` in `useNetworksRelationship`